frontend/amundsen_application/static/js/pages/TableDetailPage/TableIssues/styles.scss

```diff
@@ -7,6 +7,15 @@ $issue-banner-height: 40px;
 $shimmer-line-height: 16px;
 $shimmer-loader-lines: 1, 2;
 
+.table-issues-header {
+  display: flex;
+  justify-content: space-between;
+
+  .create-issue-error {
+    color: $rose80 !important;
```

Comment:
Agreed on trying to avoid, but I wasn't able to find another way to override the color used in the `section-title` style. Do you have any suggestions?
On the other hand, while looking up when it may be acceptable to use `!important`, I saw an example of error text color being one case (in the opinion of that writer at least) because you know that you will always want it to be that color

Reply:
That's a good point.
You can always avoid it by adding more specificity to the selector, but that doesn't always look good either.

Keep it as it is, is not a big deal!